resources/common/plugins/macro/batch-edit-oto-preutter/batch-edit-oto-preutter.js

```diff
@@ -0,0 +1,48 @@
+let selectedEntryIndexes = params["selector"]
+let preutterance = params["preutterance"]
+let overlap = params["overlap"]
+
+if (debug) {
+    console.log(`Input entries: ${entries.length}`)
+    console.log(`Selected entries: ${selectedEntryIndexes.length}`)
+}
+
+for (let index of selectedEntryIndexes) {
+    let entry = entries[index]
+    let edited = Object.assign({}, entry)
+    let offset = entry.points[3]
+    let preutter = entry.points[1] - offset
+
+    function tryParseInt(str) {
+        const num = parseInt(str, 10);
+        if (isNaN(num)) {
+            return { success: false, value: null };
+        }
+        return { success: true, value: num };
+    }
+
+    let result = tryParseInt(preutterance);
+    if (result.success) {
```

Comment:
The unsuccessful case might actual happen, so please throw an error to display readable messages to users. See: You can probably throw an error. See: https://github.com/sdercolin/vlabeler/blob/main/docs/scripting.md#error-handling